v2/cf-nodejs-logging-support/src/lib/config/config.ts

```diff
@@ -190,4 +205,21 @@ export default class Config {
 
         return index;
     }
+
+    private isReducedField(field: ConfigField): boolean {
```

Comment:
redacted vs reduced => can we use a single term? This was already confusing in existing library.

Reply:
The terms "reduced" and "redacted" are supposed to have differente meanings:

- Reduced fields:
    - Fields that have the env variable not set to true
    - Can be either envVarSwitch or envVarRedacted

- Redacted fields:
    - Fields with envVarRedacted

Maybe we can rename the function to `shouldBeReduced` to make it more clear. What do you think?